net.devolutions.RDM.yml

```diff
@@ -0,0 +1,38 @@
+app-id: net.devolutions.RDM
+runtime: org.gnome.Platform
+runtime-version: '3.38'
```

Comment:
this runtime is deprecated.

```suggestion
runtime-version: '42'
```

Do you really need the gnome runtime?

Reply:
We use gnome-keyring. I'll retest with freedesktop.

Reply:
nontheless, 3.38 is EOL